### PR TITLE
Fix/row use centralized name

### PR DESCRIPTION
**Pull request by SquarePlayn**
#### What have I done
There is a centralized function in the sheetState to get the display ID. I changed the table to use that (and added some functionality around it to keep things in the same place).

#### Testing
Automated testing. Else load 1 and/or 2 timing files and check first column

#### Addresses SRD section(s)
Table sections. SheetService. Class Diagram

### CODE REVIEW

File: src/app/core/services/sheet.service.ts [hunk 1]
```diff
@@ -193,6 +193,11 @@ export class SheetService {
         return name + '.' + timingId;
     }
 
+    getDifName(sheetState: SheetState): string {
```
Comment (Dacaspex) [inline src/app/core/services/sheet.service.ts]: Doc comment?

File: src/app/table/entry/string-entry.ts [hunk 2]
```diff
@@ -44,7 +49,12 @@ export class StringEntry implements Entry {
      * returns the value of this Entry to be displayed
      */
     public getDisplayValue(): string {
-        return this.value;
+        let padded = this.value;
```
Comment (Dacaspex) [inline src/app/table/entry/string-entry.ts]: What is happening here precisely? I think adding comment helps

Comment (SquarePlayn) [reply src/app/table/entry/string-entry.ts]: Addressed in #393

File: src/app/core/services/sheet.service.ts (hunk 1 above)
Comment (SquarePlayn) [reply src/app/core/services/sheet.service.ts]: Added doc comment

**Pull request merged**